TestProject/input/User.php

```diff
@@ -17,38 +17,54 @@
 class User extends Base
```

Comment:
Please also improve the class documentation.
Strictly speaking "command line" refers to the parameters passted to a command when it is executed. But since this input "asks" for input, the word "prompt" is more appropriate and used. So something like "The user is prompted to enter cells that are alive" would be better.